src/pages/faq.js

```diff
@@ -31,7 +32,7 @@ function Faq(props) {
         <FaqCard />
         <FaqCard />
       </FaqSection>
-    </GlobalStyles>
+      </div>
```

Comment:
Looks like your code formatter is not working correctly here. This could be an indicator that there is an issue in this file.

Reply:
Will fix this in another branch